contrib/cache_stdlibs.jl

```diff
@@ -23,13 +23,14 @@ stdlibs = [
     # 3-depth packages
     :LibGit2_jll,
 
+    # 4-depth packages
+    :SparseArrays,
+
     # 7-depth packages
     :LLD_jll,
-    :SuiteSparse_jll,
 
     # 9-depth packages
     :Statistics,
-    :SuiteSparse,
```

Comment:
Same here, as long as we ship a copy of the source with builds of Julia we should create a cache for them as well.